venues/ICLR.cc/2019/Conference/python/iclr19.py

```diff
@@ -571,10 +572,36 @@
 
 })
 
-
+# This is the Conference level Invitation for all withdrawn submissions
+withdrawn_submission_invitation = openreview.Invitation.from_json({
+    'id': WITHDRAWN_SUBMISSION_TEMPLATE_STR,
```

Comment:
I don't think this needs to be a templated invitation. I think you can have it be a global invitation like the blind submission